public/app/core/components/NestedFolderPicker/Skeleton.tsx

```diff
@@ -0,0 +1,36 @@
+import { css } from '@emotion/css';
+import Skeleton from 'react-loading-skeleton';
+
+import type { GrafanaTheme2 } from '@grafana/data';
+import { getInputStyles, useStyles2 } from '@grafana/ui';
+
+// Aim to keep the bundle-size of the as small as possible so it doesn't
```

Comment:
"bundle-size of this" maybe? seems like a typo?

Reply:
yup. I found it very difficult to find the right words for this comment!!